config/_rails_shared.yml

```diff
@@ -115,6 +115,10 @@ Layout/InitialIndentation:
   Exclude:
     - 'app/views/**/*.erb'
 
+Layout/IndentationWidth:
```

Comment:
nit: "IndentationWidth" and "InitialIndentation" are in alphabetical order in one file and not in the other.